python/lsst/ip/isr/crosstalk.py

```diff
@@ -163,15 +169,31 @@ def fromDict(cls, dictionary):
                                f"found {dictionary['metadata']['OBSTYPE']}")
 
         calib.setMetadata(dictionary['metadata'])
-        calib._detectorName = dictionary['metadata']['DETECTOR']
-        calib._detectorSerial = dictionary['metadata']['DETECTOR_SERIAL']
+        calib._detectorName = dictionary.get('detectorName',
```

Comment:
As per comment elsewhere, DETECTOR <=> detectorId, so I'm not sure if this is the way to be making this change.

Reply:
Fixed to set `_detectorName` from key `DET_NAME`.  I think the next step is to move these common keywords into a base class method, so sub-classes only need to read/write their own properties.